ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -959,6 +959,12 @@ keys:
                 route_map_out:
                   type: str
                   description: Outbound route-map name
+                rcf_in:
```

Comment:
Same as ipv4/ipv6 can we add the rcf support for neighbor as well.

Reply:
Opps.. missed it.
Added now.